test/benchmarks/benchmark.js

```diff
@@ -0,0 +1,97 @@
+const Benchmark = require('benchmark');
+const openpgp = typeof window !== 'undefined' && window.openpgp ? window.openpgp : require('../../');
+
+const wrapAsync = func => ({
+  fn: async deferred => {
+    await func().catch(onError);
+    deferred.resolve();
+  },
+  defer: true
+});
+
+const onError = err => {
+  console.error('The time benchmark tests failed by throwing the following error:');
```

Comment:
Please add a `/* eslint-disable no-console */` at the top of the file or so, to stop it from warning about this